speech/speech_samples.rb

```diff
@@ -362,6 +362,71 @@ def speech_transcribe_model_selection file_path: nil, model: nil
   # [END speech_transcribe_model_selection]
 end
 
+def speech_transcribe_multichannel audio_file_path: nil
+  # [START speech_transcribe_multichannel]
+  # audio_file_path = "path/to/audio.wav"
+
+  require "google/cloud/speech"
+
+  speech = Google::Cloud::Speech.new
+
+  config = {
+    encoding:                                :LINEAR16,
+    sample_rate_hertz:                       44100,
+    language_code:                           "en-US",
+    audio_channel_count:                     2,
+    enable_separate_recognition_per_channel: true
+  }
+
+  audio_file = File.binread audio_file_path
+  audio      = { content: audio_file }
+
+  response = speech.recognize config, audio
+
+  results = response.results
+
+  results.each_with_index do |result, i|
```

Comment:
These are a slightly different style than the other Ruby samples _(re: how they go through results/alternatives)_, but I like this a lot (and this is more _correct_, looking at the first alternative of each result) 👍

+1